data/roles/gecko_t_osx_1100_m1_loaner.yaml

```diff
@@ -9,10 +9,10 @@ worker_metadata:
   provisionerId: "%{lookup('gw.provisioner_id')}"
 
 worker:
-  taskcluster_version: 40.0.3
+  taskcluster_version: 54.4.1
```

Comment:
Can you bump this to the version we deployed most recently (55.3.0)?

Reply:
Done